src/components/interfaces/MOBILE_API.xml

```diff
@@ -2913,6 +2921,33 @@
             <description>Region</description>
         </param>
     </struct>
+
+    <struct name="StationIDNumber">
```

Comment:
Add `since="5.0"` to new structs

Reply:
@jacobkeeler Fixed at commit fd9f349304e4d47d25d1fed79a0442a599717de8